src/components/App/App.js

```diff
@@ -93,7 +98,7 @@ export default class App extends Component {
 				{!this.state.data ? <p>Loading...</p> : <p>SWAPI-box</p>}
 				{!this.state.data
 					? <p>No Cards yet</p>
-					: <CardContainer data={this.state.data} />}
+					: <CardContainer peopleArray={this.state.data[0]} />}
```

Comment:
You should take a look at the state in the React Dev tools to check out how our state is organized now. Instead of being an array of 4 Objects, the People and Planets are arrays